### ISSUE TITLE
[install fails] gcc-9 build fails for ARDB 

### ISSUE THREAD

**Issue by blueteamzone** [CIRCL/AIL-framework#496]
# ARDB Fails during installation
## ardb repo from yinqiwen doesn't seem to be maintained anymore
I am running 
#### 5.5.0-kali2-amd64 #1 SMP Debian 5.5.17-1kali1 (2020-04-21) x86_64 GNU/Linux
```
# ARDB #
test ! -d ardb/ && git clone https://github.com/yinqiwen/ardb.git
pushd ardb/
make
popd
```
I have the following gcc version when I try to compile ardb I get this error:
```
error: implicitly-declared 
‘constexpr rocksdb::FileDescriptor::FileDescriptor(const rocksdb::FileDescriptor&)’ 
is deprecated [-Werror=deprecated-copy]
```
```
gcc (Debian 9.3.0-10) 9.3.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Someone has already opened an issue over yinqiwen/ardb#484
This issue seems to related to a gcc-9 change
https://gcc.gnu.org/gcc-9/changes.html
grpc/grpc#19570
GPUOpen-Drivers/AMDVLK#131

I have also tried to build ARDB using gcc-7 following this commit fix
https://github.com/CIRCL/AIL-framework/pull/378/commits/be1bb1074d51e9cf3af3fee921cc83ea5884d0c0
However, I was unable to fix the issue even with gcc7
Are there any workarounds? Thanks

**Comment by reedcrif** [comment]
Hi, Same situation here.

I tried some little tests in order to bypass the issue. I modifying the Makefile (_ardb/src/Makefile_), line 402:


```
CXXFLAGS="-I${SNAPPY_PATH}" CFLAGS="-I${SNAPPY_PATH}" LDFLAGS="${SNAPPY_PATH}/.libs" USE_RTTI=1 $(MAKE) static_lib && \
```
by:

```
CXXFLAGS="-Wno-deprecated-copy" CFLAGS="-I${SNAPPY_PATH}" LDFLAGS="${SNAPPY_PATH}/.libs" USE_RTTI=1 $(MAKE) static_lib && \
```

Compilation continue but a lot of error appear after so I give up.

Hope a workaround will be find quickly :)

**Comment by lantuin** [comment]
I've resolved with a workaround

https://github.com/CIRCL/AIL-framework/issues/369#issuecomment-624475481

**Comment by Terrtia** [comment]
Fixed in AIL v5.0
The database has been migrated from ARDB to Kvrocks.
